src/it/correct/src/main/java/Parser.java

```diff
@@ -0,0 +1,10 @@
+import java.io.File;
+
+@SuppressWarnings({"OOP.WorkerCheck", "OOP.MutableStateCheck"})
```

Comment:
I'm not sure about the `Worker`, for programmer who has never seen this concept before, term `Worker` won't be obvious

Reply:
@l3r8yJ how to name it then?
`ErNameCheck`?

Reply:
maybe `ErSuffixCheck`